lib/mpl_toolkits/mplot3d/proj3d.py

```diff
@@ -72,25 +72,33 @@ def rotation_about_vector(v, angle):
     return R
 
 
-def view_transformation(E, R, V, roll):
-    n = (E - R)
-    n = n/np.linalg.norm(n)
-    u = np.cross(V, n)
+def view_axes(E, R, V, roll):
+    """
+    Get the unit viewing axes in data coordinates.
+    `u` is towards the right of the screen
+    `v` is towards the top of the screen
+    `w` is out of the screen
+    """
+    w = (E - R)
+    w = w/np.linalg.norm(w)
+    u = np.cross(V, w)
     u = u/np.linalg.norm(u)
-    v = np.cross(n, u)  # Will be a unit vector
+    v = np.cross(w, u)  # Will be a unit vector
 
     # Save some computation for the default roll=0
     if roll != 0:
         # A positive rotation of the camera is a negative rotation of the world
-        Rroll = rotation_about_vector(n, -roll)
+        Rroll = rotation_about_vector(w, -roll)
         u = np.dot(Rroll, u)
         v = np.dot(Rroll, v)
+    return u, v, w
 
+
+def view_transformation(u, v, w, E):
```

Comment:
See the previous comment about renaming the parameters of `view_transformation` and needing a deprecation or alternative path for it.